.github/pull_request_template.md

```diff
@@ -31,6 +31,20 @@ This save file can be used to verify the bugfix. The bug will occur when using {
 {{attach a save file that can be used to verify your bugfix. It MUST have no plugin requirements}}
 
 
+-----------------------
+**Balance:**
+
+## Changes Made
+{{add details}}
```

Comment:
this should be expanded upon imo

Reply:
```suggestion
{{concisely explain what changes you made to improve balance}}
```
What about this?

Reply:
Maybe just `concisely explain the changes you made`? The part about improving balance is what the "Reasoning" section is for, that's where you explain why the changes are improving balance.

Reply:
`summarize all the changes you made` maybe?

Reply:
I'd say @quyykk's suggestion.